manim/scene/scene_file_writer.py

```diff
@@ -608,21 +616,12 @@ def flush_cache_directory(self):
         for f in cached_partial_movies:
             os.remove(f)
         logger.info(
-            f"Cache flushed. {len(cached_partial_movies)} file(s) deleted in {self.partial_movie_directory}."
+            f"Cache flushed. {len(cached_partial_movies)} file(s) deleted in %(par_dir)s.",
+            {"par_dir": self.partial_movie_directory},
```

Comment:
Same thing about trailing comma (x6 in this file)

Reply:
Yes, this is super annoying. I will see what I can do

Reply:
I think they will all go away if you just remove the trailing comma and run black again.